airflow/providers/apache/kafka/operators/consume.py

```diff
@@ -0,0 +1,184 @@
+        defaults to None
+    :param commit_cadence: When consumers should commit offsets ("never", "end_of_batch","end_of_operator"),
+        defaults to "end_of_operator";
+        if end_of_operator, the commit() is called based on the max_messages arg. Commits are made after the
+        operator has processed the apply_function method for the maximum messages in the operator.
+        if end_of_batch, the commit() is called based on the max_batch_size arg. Commits are made after each
+        batch has processed by the apply_function method for all messages in the batch.
+        if never,  close() is called without calling the commit() method.
+    :param max_messages: The maximum total number of messages an operator should read from Kafka,
+        defaults to None
+    :param max_batch_size: The maximum number of messages a consumer should read when polling,
+        defaults to 1000
+    :param poll_timeout: How long the Kafka consumer should wait before determining no more messages are
+        available, defaults to 60
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:ConsumeFromTopicOperator`
+    """
+
+    BLUE = "#ffefeb"
+    ui_color = BLUE
+    template_fields = (
+        "topics",
+        "apply_function",
+        "apply_function_args",
+        "apply_function_kwargs",
+        "kafka_config_id",
+    )
+
+    def __init__(
+        self,
+        topics: str | Sequence[str],
+        kafka_config_id: str = "kafka_default",
+        apply_function: Callable[..., Any] | str | None = None,
+        apply_function_batch: Callable[..., Any] | str | None = None,
+        apply_function_args: Sequence[Any] | None = None,
+        apply_function_kwargs: dict[Any, Any] | None = None,
+        commit_cadence: str | None = "end_of_operator",
+        max_messages: int | None = None,
+        max_batch_size: int = 1000,
+        poll_timeout: float = 60,
+        **kwargs: Any,
+    ) -> None:
+
+        super().__init__(**kwargs)
+
+        self.topics = topics
+        self.apply_function = apply_function
+        self.apply_function_batch = apply_function_batch
+        self.apply_function_args = apply_function_args or ()
+        self.apply_function_kwargs = apply_function_kwargs or {}
+        self.kafka_config_id = kafka_config_id
+        self.commit_cadence = commit_cadence
+        self.max_messages = max_messages or True
+        self.max_batch_size = max_batch_size
+        self.poll_timeout = poll_timeout
+
+        if self.commit_cadence not in VALID_COMMIT_CADENCE:
+            raise AirflowException(
+                f"commit_cadence must be one of {VALID_COMMIT_CADENCE}. Got {self.commit_cadence}"
+            )
+
+        if self.max_messages and self.max_batch_size > self.max_messages:
+            self.log.warning(
+                "max_batch_size (%s) > max_messages" " (%s). Setting max_messages to" " %s ",
```

Comment:
```suggestion
                "max_batch_size (%s) > max_messages (%s). Setting max_messages to %s ",
```
Probably some funny business from `black` re-formatting.